include/seqan3/alignment/pairwise/align_pairwise.hpp

```diff
@@ -187,7 +188,7 @@ constexpr auto align_pairwise(sequence_t && sequences,
     auto select_execution_handler = [&] ()
     {
         if constexpr (std::same_as<execution_handler_t, detail::execution_handler_parallel>)
-            return execution_handler_t{get<align_cfg::parallel>(complete_config).value};
+            return execution_handler_t{get<align_cfg::parallel>(complete_config).thread_count.value()};
         else
             return execution_handler_t{};
```

Comment:
```suggestion
        {
            if (!get<align_cfg::parallel>(complete_config).thread_count.has_value())
                throw std::runtime_error{"You must configure the number of threads in seqan3::align_cfg::parallel."};
            return execution_handler_t{get<align_cfg::parallel>(complete_config).thread_count.value()};
        }
        else
        {
            return execution_handler_t{};
        }
```

I think this will be more easy to understand for the user than a `std::bad_optional_acces` ?
Maybe rephrase the sentence, it's just a proposal :)

Reply:
Yes, I like this suggestion :)